schema/Readme.md

```diff
@@ -507,7 +507,7 @@ we'll use `primary key(blog, writer)` to ensure that there can't be no more than
 For implicit many-to-many relations, you **must** include both `Blog.authors` and `Writer.blogs`. If one of these fields is missing, Prisma will assume it's a
 **One-to-Many (1:N)** relationship.
 
-##### Explicit Many-to-Many (M:N) Relationships
+##### Explicit Many-To-Many (M:N) Relationships With Or Without Extra Columns
```

Comment:
inconsistent with previous headline

Reply:
What do you mean?

Reply:
![image](https://user-images.githubusercontent.com/183673/76335071-4822bd80-62f4-11ea-9cb4-957d38e6c320.png)

Reply:
Do you have a suggestion for improving this? Extra columns doesn't apply to implicit m:n relation in my understanding.

Reply:
The previous headline has "Many-to-Many".
You changed this one to be "Many-To-Many".
Not the lowercase vs. the Uppercase `t`.

Reply:
aha! okay, that is descriptive, let me make all titles to be in title case! (as per VSCode) unless we have a special way to represent titles?

Reply:
The existing way to write Many-to-Many was perfectly fine.

Reply:
Fixed